src/utils/colors.js

```diff
@@ -0,0 +1,19 @@
+function getHexCodeWithTransparency(colorCode, opacity) {
```

Comment:
Could this be done with chroma?

Reply:
Yes, it could. Thanks for tip. See the next commit.